### PR TITLE
GUI-1679 simplified server-side object delete using bucket.delete_keys()

### PR DESCRIPTION
**Pull request by dkavanagh**
simplified server-side object delete using bucket.delete_keys()

https://eucalyptus.atlassian.net/browse/GUI-1679

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/234/
Test FAILed.

**Comment by kamalgill** [comment]
@dkavanagh: PR looks good, but we probably no longer need the progress bar for delete operations since we're using one API call now, right?

File: eucaconsole/views/buckets.py [hunk 1]
```diff
@@ -204,21 +204,13 @@ def bucket_delete_keys(self):
         errors = []
```
Comment (kamalgill) [inline eucaconsole/views/buckets.py]: This variable is unused and can be safely removed.

**Comment by euca-qa-gui-tester** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/236/
Test FAILed.

**Pull request merged**